libs/api/src/main/java/de/cyface/api/model/ClassifiedSegment.java

```diff
@@ -201,34 +231,65 @@ public void asGeoJson(final Consumer<String> handler) {
         handler.accept(coordinates);
         handler.accept("},");
 
-        final var builder = new Json.JsonObject.Builder();
-        addTag(getTags(), "name", builder);
-        addTag(getTags(), "highway", builder);
-        addTag(getTags(), "surface", builder);
-        final var tagsObject = builder.build();
+        final var properties = propertiesAsGeoJson(handler);
+        handler.accept(jsonKeyValue("properties", properties).getStringValue());
+
+        handler.accept("}");
+    }
+
+    /**
+     * Converts the properties of this measurement as a list of {@link Json.KeyValuePair}s.
+     *
+     * @return the converted properties
+     */
+    protected List<Json.KeyValuePair> properties() {
```

Comment:
Had to change this a little bit so that `ClassifiedMeasurementSegment` can extend those methods and inject measurement id and device id.